assetman/tools.py

```diff
@@ -65,14 +73,10 @@ def get_parser(template_path, template_type, settings):
     """ Factory method to return appropriate parser class """
     #TODO: dynamic import / return based on settings / config
     #avoids circular dep
-    assert template_type in ["tornado_template", "django_template"]
-    assert isinstance(template_path, (str, unicode))
+    assert template_type in ["tornado_template"]
+    assert isinstance(template_path, str)
     logging.info('parsing for %s %s', template_type, template_path)
     if settings.get('verbose'):
-        print template_type, template_path
-    if template_type == "tornado_template":
-        from assetman.parsers.tornado_parser import TornadoParser
-        return TornadoParser(template_path, settings)
-    else:
-        from assetman.parsers.django_parser import DjangoParser
-        return DjangoParser(template_path, settings)
+        print(template_type, template_path)
+    from assetman.parsers.tornado_parser import TornadoParser
```

Comment:
Can we move this import to the top now since we don't need to dynamically import?

Reply:
Seems like the dynamic import might be there because there is a circular dependency between `assetman/tools.py` and `assetman/parsers/tornado_parser.py`